numpy/core/_ufunc_config.py

```diff
@@ -182,13 +182,25 @@ def geterr():
 @set_module('numpy')
 def setbufsize(size):
     """
-    Set the size of the buffer used in ufuncs.
+    Set the size of the buffer used in :ref:`ufuncs`.
 
     Parameters
     ----------
     size : int
-        Size of buffer.
+        Size of buffer. Must be a multiple of 16.
 
+    Returns
+    -------
+    setbufsize : int
+        Original size of ufunc buffer in bytes.
+
+    Examples
+    --------
+    >>> original_size = np.getbufsize()
+    >>> np.setbufsize(4096) == original_size
+    True
+    >>> np.getbufsize()
+    4096
```

Comment:
```suggestion
    >>> original_size = np.getbufsize()
    >>> np.setbufsize(4096)
    >>> np.getbufsize()
    4096
    >>> np.setbufsize(original_size)
```

Reply:
The `==` test doesn't add anything and is potentially confusing. Also resets the size as @mhvk suggested.